### PR TITLE
models/VP2420.yml: added

### PR DESCRIPTION
**Pull request by filipleple**

### CODE REVIEW

**Comment by macpijan** [comment]
Thanks @filipleple , but please fix pre-commit.

File: osfv_cli/osfv_cli/models/VP2420.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+flash_chip:
+  # flash chip model to be used by flashrom (optional - if not given, no chip
```
Comment (macpijan) [inline osfv_cli/osfv_cli/models/VP2420.yml:3]: I know we have these comments in others, they are not required anymore, we have some documentation already. It is better to document them in one place.

**Comment by filipleple** [comment]
@macpijan [removed](https://github.com/Dasharo/osfv-scripts/compare/aa1d96d25d7ee08e54ec330dc87707f3663c553f..1bad047a38a71d40202383c4e38c3f51fb847aa0) all comments and the trailing whitespace. Not sure why pre-commit fails now - locally it's satisfied, here it fails due to `osfv_cli.py`, which I'm not modifying

**Comment by macpijan** [comment]
@filipleple If yo uwant to find out, you can easily run pre-commit locally: `pre-commit run --all-files`.

It is probably my fault here, as the check in master fails as well. But if you can fix it in this PR, I won't complain :)

**Comment by filipleple** [comment]
@macpijan that was a good tip, all green now

**Review by macpijan — APPROVED** [review]

**Pull request merged**